onnxruntime/core/graph/graph.cc

```diff
@@ -4221,30 +4255,46 @@ Status Graph::InlineIfSubgraph(bool condition_value, Node& if_node, const loggin
       }
     }
 
-    const auto new_node_name = GenerateNodeName(make_unique(node->Name()));
+    const auto new_node_name = GenerateNodeName(make_unique(node->OpType()));
     Node& new_node = AddNode(new_node_name, node->OpType(), node->Description(),
                              new_node_input_defs,
                              new_node_output_defs,
                              nullptr,
                              node->Domain());
 
+    if (!is_this_main_graph) {
+      int arg_pos = -1;
+      for (auto* input_def : new_node_input_defs) {
```

Comment:
Is this intended to be different from `map_defs` above, or can this be replaced by a call to `map_defs`? At least the check for `Exists(nodearg)` seems potentially relevant here.